libminifi/include/EventDrivenSchedulingAgent.h

```diff
@@ -54,23 +45,16 @@ class EventDrivenSchedulingAgent : public ThreadedSchedulingAgent {
     time_slice_ = std::chrono::milliseconds(slice);
   }
 
+  EventDrivenSchedulingAgent(const EventDrivenSchedulingAgent &parent) = delete;
+  EventDrivenSchedulingAgent& operator=(const EventDrivenSchedulingAgent &parent) = delete;
```

Comment:
`SchedulingAgent` already seem to have these disabled

Reply:
:+1: You are right, removed from all the child class in https://github.com/apache/nifi-minifi-cpp/pull/1474/commits/d263e1b23eeb4aa21cf3a0a9c8eb22d413af6cee